src/devices/Rtc/Devices/Pcf8563/Pcf8563.cs

```diff
@@ -2,9 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Iot.Device.Common;
```

Comment:
unnecessary change. This is only ok if you're doing any other functional changes

Reply:
It's using NumberHelper, which is in Iot.Device.Common.